03/index.html

```diff
@@ -11,6 +11,20 @@
     <main class="t">T</main>
     <header class="p">P</header>
     <side class="v">V</side>
-
+    <body>
```

Comment:
Tutaj mamy 2x `body` - myślę, że przez pomyłkę zostawiłaś niepotrzebnie to co jest powyżej wraz z body.